python/smqtk/representation/key_value/postgres.py

```diff
@@ -206,7 +210,7 @@ def _bin_to_py(b):
         :rtype: collections.Hashable
 
         """
-        return pickle.loads(str(b))
+        return pickle.loads(bytes(b))
```

Comment:
We talked offline.  This is a non-issue because both keys and values are both allowed to be arbitrary python types (with a ``collections.Hashable`` restriction on the keys type) and need to serialized and deserialized in some manner when going into and out of the database.  Due to the breadth of python classes/objects that may be used here, pickle currently seems to be the only sensible choice.

Additionally, it seemed that there was a miscommunication of the purpose of this method.  I will add a commit that should clarity the input and output intents for this method.